conans/client/cmd/search.py

```diff
@@ -70,7 +66,7 @@ def _search_packages_in_local(self, ref=None, query=None, outdated=False):
 
         if outdated:
             ordered_packages = filter_outdated(ordered_packages, recipe_hash)
```

Comment:
I want to understand what "outdated" means now with revisions. Apparently nothing, but at least some tests should break. I think we want to remove this.